### PR TITLE
Added support for setting bottom padding

### PR DESCRIPTION
**Pull request by yokoboko**
Tested under iOS and Android

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 2.3.4
+* Added  support for setting bottom padding
```
Comment (Zazo032) [inline CHANGELOG.md]:
```suggestion
* Added support for setting bottom padding
```

File: lib/intercom_flutter.dart [hunk 2]
```diff
@@ -28,6 +28,10 @@ class Intercom {
     return _unreadChannel.receiveBroadcastStream();
   }
 
+  static Future<dynamic> setBottomPadding(int padding) {
```
Comment (Zazo032) [inline lib/intercom_flutter.dart:35]: Could you add the docs for this new method, please? Also, what happens if `padding` is `null`? Is it a valid value for the native SDK? I see it's avoided on Android, but not on iOS

**Review by Zazo032 — APPROVED** [review]
Hi @yokoboko, thanks for the PR 👍 

Overall LGTM with minor nits regarding docs and nullability of the `padding`

File: lib/intercom_flutter.dart (hunk 2 above)
Comment (yokoboko) [reply lib/intercom_flutter.dart:35]: Thank you for the review. I implemented these wanted changes to new commit and tested on iOS with null padding.

Comment (Zazo032) [reply lib/intercom_flutter.dart:35]: Thanks!

**Pull request merged**